src/Compiler/Microsoft.AspNetCore.Razor.Language/legacyTest/Legacy/TagHelperParseTreeRewriterTest.cs

```diff
@@ -536,7 +537,7 @@ public void UnderstandsAllowedChildren1()
     {
         // Arrange
         var documentContent = "<p><br /></p>";
-        var descriptors = TagHelperParseTreeRewriterTest.GetAllowedChildrenTagHelperDescriptors(new[] { "br" });
+        var descriptors = GetAllowedChildrenTagHelperDescriptors(["br"]);
```

Comment:
I updated the entire file for any compiler tests that were affected by this change, but I didn't survey all compiler, since that's kind of out of scope for this PR. Did you spot tests I missed in the test files I modified?

Reply:
> Did you spot tests I missed in the test files I modified?

Yes, couple of tests below this one where I commented.

Reply:
Will take a look. Thanks!

Reply:
@jjonescz: I just scanned through this file and did not see any test that weren't updated to match. I must be missing something.

Reply:
This file looks good to me though, so I'm going to go ahead and merge.

Reply:
https://github.com/dotnet/razor/pull/9947